libraries/AC_AttitudeControl/AC_AttitudeControl.h

```diff
@@ -295,7 +296,7 @@ class AC_AttitudeControl {
     // calculates the velocity correction from an angle error. The angular velocity has acceleration and
     // deceleration limits including basic jerk limiting using smoothing_gain
```

Comment:
The comment mentions "smoothing_gain" but this argument has been renamed to "input_tc" now.